pymc/sampling/jax.py

```diff
@@ -568,6 +561,15 @@ def sample_jax_nuts(
             DeprecationWarning,
         )
 
+    if postprocessing_backend is not None:
+        import warnings
+
+        warnings.warn(
+            "postprocessing_backend={'cpu', 'gpu'} will be removed in a future release, "
```

Comment:
We should deprecate before rendering the argument useless or raise already. Also can the message mention the alternative is num_chunks now?

Reply:
Ok, makes sense - I can add back that functionality - it's just a few extra branches to keep track of

Reply:
Added back logic for `postprocessing_backend`.  It doesn't have any integration with chunked sampling but restores postprocessing all at once on a different backend.